study-builder/studies/shared/family-history/family-history-top-level.conf

```diff
@@ -11,6 +11,8 @@
   INSTRUCTIONS_BLOCK_SELF_EN={include required("snippets/family-history-section-instructions.conf")
     INSTRUCTIONS_P1_EN="""
               In this survey we would like to know the <span class="Semibold">living status, age, and cancer history</span> of people in your <span class="Semibold">biological, or blood-related, family.</span> We recognize that there are many different types of families, so please skip sections that do not apply to your family tree.""",
+    INSTRUCTIONS_P1_ES="""
+                  In this survey we would like to know the <span class="Semibold">living status, age, and cancer history</span> of people in your <span class="Semibold">biological, or blood-related, family.</span> We recognize that there are many different types of families, so please skip sections that do not apply to your family tree.""",
```

Comment:
https://github.com/broadinstitute/ddp-study-server/blob/32721a98c7f4313e7a8d29aa98cd6a3e9f8d6f53/study-builder/studies/pancan/family-history/family-history-top-level.conf#L12

This looks like a similar block, but for `pancan` specifically